src/sources/file.rs

```diff
@@ -144,8 +144,10 @@ pub struct FileConfig {
 
     /// The directory used to persist file checkpoint positions.
     ///
-    /// By default, the [global `data_dir` option][global_data_dir] is used. Make sure the running user has write
-    /// permissions to this directory.
+    /// By default, the [global `data_dir` option][global_data_dir] is used.
+    /// Make sure the running user has write permissions to this directory.
+    ///
+    /// If this directory is specified, then it will be created under the [global `data_dir`][global_data_dir].
```

Comment:
Is this true? The example has a fully-qualified path. In that case does it still end up under the global directory? Maybe we should show a relative path example instead?

Reply:
Yes, that is how it works. See [code here](https://github.com/vectordotdev/vector/blob/b1502ec185a517f2c95078f5a70acae7baaf1c30/src/sources/file.rs#L415-L422). It is good to work this way but the docs were misleading.